intbot/core/integrations/github.py

```diff
@@ -1,17 +1,35 @@
-import dataclasses
+"""
+The Github Integration is mostly about parsing Github Webhooks.
+
+This is split into three parts.
+
+1. Save the webhook in the database (not in this file)
+2. Prep webhook - download extra data that's missing in the webhook
+3. Proces webhook - usually means create a discord message with a notification
+"""
+
+from typing import Any
 
 import httpx
+from core.models import Webhook
 from django.conf import settings
+from pydantic import BaseModel
```

Comment:
Please run `isort`, this triggers me 😅

Reply:
The ruff format with isort is run on this, but I think the path to the module is not recognised as firstparty. We can fix that in a spearate PR see #11